brownie/project/compiler.py

```diff
@@ -505,12 +528,16 @@ def expand_source_map(source_map):
     return source_map
 
 
-def _expand_row(row):
-    row = row.split(':')
-    return [int(i) if i else None for i in row[:3]] + row[3:] + [None] * (4 - len(row))
+def _expand_row(row_str: str) -> Any:
+    row = row_str.split(':')
+    # ignore typing of next line
```

Comment:
Agreed, this is ugly.  I had to generate similar code for Vyper's source map and it looks a lot cleaner: https://github.com/ethereum/vyper/blob/faacf1503e02b7af935a6dcf8eda4ec697ed2101/vyper/compiler.py#L177-L196

Feel free to remove this comment and raise another issue about how ugly my code golf looks :rofl: